ADJ-Internship/Repository/Implementations/PurchaseOrderRepository.cs

```diff
@@ -17,6 +19,26 @@ public PurchaseOrderRepository(ApplicationDbContext dbContext) : base(dbContext)
         }
 
         protected override Func<IQueryable<PurchaseOrder>, IQueryable<PurchaseOrder>> IncludeDependents =>
-            po => po.Include(x => x.Items);
+            po => po.Include(x => x.Items);  
+    }
+
+    public class OrderRepository : RepositoryBase<Order>, IOrderRepository
+    {
+        public OrderRepository(ApplicationDbContext dbContext) : base(dbContext)
+        {
+        }
+
+        protected override Func<IQueryable<Order>, IQueryable<Order>> IncludeDependents =>
+            po => po.Include(x => x.orderDetails);
+    }
+
+    public class OrderDetailRepository : RepositoryBase<OrderDetail>, IOrderDetailRepository
+    {
+        public OrderDetailRepository(ApplicationDbContext dbContext) : base(dbContext)
+        {
+        }
+
+        protected override Func<IQueryable<OrderDetail>, IQueryable<OrderDetail>> IncludeDependents =>
```

Comment:
include Order or OrderId?

Reply:
fix by changing dependence to null, since OrderDetail doesn't have any dependents